package.json

```diff
@@ -86,7 +86,7 @@
     "react-native-screens": "^3.29.0",
     "react-native-screenshot-prevent": "^1.1.9",
     "react-native-ssl-public-key-pinning": "^1.1.3",
-    "react-native-svg": "^14.1.0",
+    "react-native-svg": "13.4.0",
```

Comment:
Wait! Why downgrading?

Reply:
I had to downgrade this version to fix an issue with the appearance of the loading screen using a light background.

Reply:
@rodrigoncalves can you comment or add screenshot/link to show what issue is being caused by downgrading this library?

Reply:
https://stackoverflow.com/a/76749105

<img src="https://github.com/rsksmart/rif-wallet/assets/4074139/b8ca2124-c7c8-42ee-983f-2dc69f757450" width="200" />